dev/dependencyList

```diff
@@ -30,8 +30,16 @@ htrace-core4/4.1.0-incubating//htrace-core4-4.1.0-incubating.jar
 jackson-annotations/2.11.4//jackson-annotations-2.11.4.jar
 jackson-core/2.11.4//jackson-core-2.11.4.jar
 jackson-databind/2.11.4//jackson-databind-2.11.4.jar
+jakarta.servlet-api/4.0.4//jakarta.servlet-api-4.0.4.jar
 javassist/3.18.1-GA//javassist-3.18.1-GA.jar
 jcl-over-slf4j/1.7.30//jcl-over-slf4j-1.7.30.jar
+jetty-http/9.4.40.v20210413//jetty-http-9.4.40.v20210413.jar
```

Comment:
why not jetty 11?

Reply:
Jetty 11 required JVM 11+, see https://www.eclipse.org/jetty/download.php